packages/open-payments/src/client/requests.ts

```diff
@@ -129,6 +134,58 @@ export const post = async <TRequest, TResponse>(
   }
 }
 
+export const deleteRequest = async <TResponse>(
+  deps: BaseDeps,
+  args: DeleteArgs,
+  openApiResponseValidator: ResponseValidator<TResponse>
+): Promise<void> => {
+  const { axiosInstance, logger } = deps
+  const { accessToken } = args
+
+  const requestUrl = new URL(args.url)
+  if (process.env.NODE_ENV === 'development') {
+    requestUrl.protocol = 'http'
+  }
+
+  const url = requestUrl.href
+
+  try {
+    const { data, status } = await axiosInstance.delete<TResponse>(url, {
+      headers: accessToken
+        ? {
+            Authorization: `GNAP ${accessToken}`
+          }
+        : {}
+    })
+
+    try {
+      openApiResponseValidator({
+        status,
+        body: data || undefined
```

Comment:
<img width="927" alt="Screen Shot 2022-12-14 at 7 21 16 PM" src="https://user-images.githubusercontent.com/15069181/207676329-1485828a-fddb-422f-8505-b3e4b88da3b7.png">

So it looks like for the grant cancellation request which doesn't have a response body, the validator will support `null | undefined`. When I was testing, `nock` would insert empty string as a the response body, so I'm effectively allowing the AS to return empty string for an empty response here.